carma/launch/guidance.launch.py

```diff
@@ -316,6 +316,59 @@ def generate_launch_description():
         ]
     )
 
+    trajectory_follower_container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='trajectory_follower_container',
+        executable='carma_component_container_mt',
+        namespace=GetCurrentNamespace(),
+        composable_node_descriptions=[
+            ComposableNode(
+                package='trajectory_follower_nodes',
+                plugin='autoware::motion::control::trajectory_follower_nodes::LatLonMuxer',
+                name='latlon_muxer_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': False},
+                    {'--log-level' : GetLogLevel('latlon_muxer', env_log_levels) }
+                ],
+                parameters=[
+                    {'timeout_thr_sec':0.5}
+                ]
+            ),
+            ComposableNode(
+                package='trajectory_follower_nodes',
+                plugin='autoware::motion::control::trajectory_follower_nodes::LateralController',
+                name='lateral_controller_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': True},
+                    {'--log-level' : GetLogLevel('lateral_controller', env_log_levels) }
+                ],
+                remappings = [
+                      ("output/lateral/control_cmd", "input/lateral/control_cmd")
```

Comment:
I think it might be better if we drop the `input` `output` naming convention for the final name resulting from the mapping. for example, consider revising to:
```
("output/lateral/control_cmd", "lateral/control_cmd")
```
(above suggestion is assuming the final topic name includes the node's name if I remember correctly. I am imagining it should look like `/guidance/lateral_controller_node/lateral/control_cmd`)

This is mainly because the name "input" would be confusing if we inspect this specific node (instead of the main trajectory_follower) because it is still output for this one.

Reply:
These are not topic names that I came up with. They are Autoware's nodes expected topics that are published by one, and consumed by another one. And I tried to minimize changes to Autoware nodes in case we want to update the package.

Reply:
I agree, the input/output naming here is a little confusing. As long as we only remapping the topic names in the launch, it should still be the same effort to update the package?

Reply:
@saina-ramyar yeah we don't need to change names in the autoware package itself. Just topic remapping for general naming here in the launch file would be good. Since we are already topic remapping from "output" to "input", we should rename it to something general.
```
"input/lateral/control_cmd" would be sth like "lateral_controller_node/lateral/control_cmd" 
```